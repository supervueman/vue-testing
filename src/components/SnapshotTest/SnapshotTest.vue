<template>
  <div>
    <h1>Snapshot test!!!</h1>
  </div>
</template>

<script>
export default {
  name: 'SnapshotTest'
}
</script>
