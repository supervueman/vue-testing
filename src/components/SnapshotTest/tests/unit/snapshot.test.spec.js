import { mount } from '@vue/test-utils'
import SnapshotTest from '@/components/SnapshotTest/SnapshotTest.vue'

describe('SnapshotTest', () => {
  it('Тест снимками', () => {
    const wrapper = mount(SnapshotTest)

    expect(wrapper.element).toMatchSnapshot()
  })
})
