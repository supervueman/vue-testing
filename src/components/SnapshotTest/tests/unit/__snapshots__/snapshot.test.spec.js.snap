// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest Тест снимками 1`] = `
<div>
  <h1>
    Snapshot test!!!
  </h1>
</div>
`;
